content/app/github/wrappers/line-element-wrapper.js

```diff
@@ -24,10 +61,14 @@ define("content/app/github/wrappers/line-element-wrapper", function(require, exp
 				var lines = [];
 				for (var i = 0; i < domElements.length; i++) {
 					var domElement = domElements[i];
-					// TODO: Consider additional filtering.
-					lines.push(new LineElementWrapper(domElement));
+					var className = domElement.className;
+					if (className === LineElementWrapper.TAG_LINE_CLASS || className === LineElementWrapper.TAG_EXPANDABLE_BUTTON_CLASS) {
+						lines.push(new LineElementWrapper(domElement));
+					}
 				}
 
+				this._populateMissingRanges(lines);
```

Comment:
Populate standard ranges is not necessary as line number are parsed inside LineElementWrapper. And code above is doing "parsing" it is creating appropriate wrappers for dom elements. You'll see in next iteration.
And populate ranges method renamed. Thanks for suggestion.